.github/workflows/release.yml

```diff
@@ -214,7 +233,7 @@ jobs:
 
       - name: Build Tauri Apps
         id: build
-        uses: tauri-apps/tauri-action@dev
```

Comment:
😮

Reply:
Moving toward a stable action? Is there a reason to stick to dev?